motion-1/Overview.bs

```diff
@@ -163,19 +235,235 @@ Media: visual
 Animatable: yes
 </pre>
 
-A distance that describes the position along the specified <a>motion path</a>.
+Describes the position of the element along the specified <a>path</a>.
+
+<dl dfn-for="offset-distance" dfn-type="value">
+<dt><<length-percentage>></dt>
+<dd>The distance from the initial position of the shape or path to the position 
+of the element.
+
+Percentages are relative to the length of the path that is the distance between 
+the initial position and the end position of the path.</dd>
+
+<dt><<size>></dt>
+<dd>Decides a point used for measuring the distance from the initial position 
+	of the path.
```

Comment:
Does this point override the end of the path for the purpose of percentage calculations? It's not clear what it does from the prose.